tensorflow/python/data/experimental/ops/threading_options.py

```diff
@@ -52,6 +54,15 @@ class ThreadingOptions(options.OptionsBase):
       "The value 0 can be used to indicate that the threadpool size should be "
       "determined at runtime based on the number of available CPU cores.")
 
+  def _get_option_attrs(self):
+    return ["max_intra_op_parallelism", "private_threadpool_size"]
```

Comment:
You can avoid introducing this method by modifying the `_has_non_default_values` to do the following instead:

```
for attr in filter(lambda x: not x.startswith("_"), dir(self)):
  ...
```

Reply:
addressed in the commit